source/exe/process_wide.cc

```diff
@@ -58,7 +57,11 @@ ProcessWide::~ProcessWide() {
 
   ASSERT(init_data.count_ > 0);
   if (--init_data.count_ == 0) {
-    ares_library_cleanup();
+    // Cleanup c-ares library if it is linked in.
```

Comment:
Add a comment that it self-inits on demand in a thread-safe way, thus avoiding initializing it if not used.

Reply:
ACK